import java.util.Scanner;

public class question14 {
    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        System.out.print("Enter a number: ");
        int num1 = input.nextInt();

        System.out.print("Enter a number: ");
        int num2 = input.nextInt();

        System.out.print("Enter a number: ");
        int num3 = input.nextInt();

        int max;

        if (num1 > num2 && num1 > num3) {
            max = num1;
        } else if (num2 > num1 && num2 > num3) {
            max = num2;
        } else {
            max = num3;
        }

        System.out.println("Largest number is: " + max);
    }
}